fastlane/metadata/android/fr-CA/full_description.txt

```diff
@@ -1,37 +0,0 @@
-WordPress pour Android vous donne accès à la puissance de la publication sur le Web. Il s’agit d’un créateur de sites Web et plus encore !
```

Comment:
Deletion of all those `fastlane/metadata/android/*/full_description.txt` files is because they were all >4000 characters (which is the limit accepted by the PlayStore) and never matched the copy that was used in the PlayStore anyway.

After fixing the missing `max_size: 4000` in our tooling (see Fastfile diff above) and re-running the lane, that took care of removing all the files which exceeded the accepted size by PlayStore, hence their deletion here.

The short and full description for the WordPress app will soon get a refresher anyway (ref: pbvYe0-JV-p2) so that's not really a big deal anyway.